docs/rp2/quickref.rst

```diff
@@ -112,14 +142,17 @@ Use the ``machine.PWM`` class::
 ADC (analog to digital conversion)
 ----------------------------------
 
-How does the ADC module work?
+RP2040 has five ADC channels in total, four of which are 12-bit SAR based
+ADCs: GP26, GP27, GP28 and GP29. The input signal for ADC0, ADC1 and ADC2 can
+be connected with GP26, GP27, GP28. The standard ADC range is 0-3.3V. The fifth channel 
```

Comment:
I think it's possible to use ADC3 with GP29.  Or did you find information that says it won't work?

Reply:
Include ADC3/GP29 in this list but add a note that "on Pico GP29 is connected to VSYS"